grunt_tasks/webpack.config.js

```diff
@@ -25,7 +25,6 @@ var webpack = require('webpack');
 var ExtractTextPlugin = require('extract-text-webpack-plugin');
 
 var paths = require('./webpack.paths.js');
-var es2015Preset = require.resolve('babel-preset-es2015');
```

Comment:
Rather than bog down this PR with any of the symlink issues, I've reverted that change and added a code comment.

We can test and revisit in the future.